content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -301,6 +301,16 @@ Below you can find installation instructions for some popular Pod network plugin
 
 {{< tabs name="tabs-pod-install" >}}
 
+{{% tab name="Antrea" %}}
+[Antrea](https://github.com/vmware-tanzu/antrea) is an [Open vSwitch](https://www.openvswitch.org/) based networking and network policy solution. Open vSwitch enables Antrea to implement Kubernetes Network Policies in a high-performance and efficient manner.
+In order Antrea to detect POD network range, you must pass `--pod-network-cidr` parameter to `kubeadm init`.
```

Comment:
This line isn't idiomatic English.

Maybe:
```suggestion
You must pass `--pod-network-cidr` parameter to `kubeadm init`. Antrea relies on this configuration.
```

(the thing I'm least keen on is “POD”, which is correct in terms of language but not right in terms of page style).

Reply:
One reason we're keen on readable English is to reduce the effort for localization teams when picking up changes.